pkg/transforms/mqtt.go

```diff
@@ -52,52 +55,58 @@ type KeyCertPair struct {
 }
 
 type MQTTSender struct {
-	client         MQTT.Client
+	client         messaging.MessageClient
```

Comment:
This is MQTT Export, not message bus. Undo changes to this file

Reply:
Just to be clear, we want to only use the MQTT functionality from `go-mod-messaging` in the SDK and we want to continue to use the client and functionality from `github.com/eclipse/paho.mqtt.golang` for the export features?

Reply:
Correct. They are separate functionality. 

Update to latest `go-mod-messaging` to enable use of MQTT for EdgeX message bus. MQTT Export remains as is since it is not related to the EdgeX message bus. @jbonafide623 fixed the missing call to Connect, so all you should need to do is update the go.mod file. 

I created an issue against  app-service-configurable to use MQTT for message bus in a new profile. See https://github.com/edgexfoundry/app-service-configurable/issues/54 . This will be the final integration point where you can actually integration tests use of MQTT message bus from Core => App Service.

Reply:
Ok, I was misunderstanding the original issue and thought the issue was that we wanted to solely use `go-mod-messaging` for MQTT messaging. After looking at this again, I think all of my changes are not needed and we only need to update the version of `go-mod-messaging` to the latest since we are using the client for messagebus support:
https://github.com/edgexfoundry/app-functions-sdk-go/blob/b5a07d60f827f2d5781e28fb3efa304f8b87fe90/internal/trigger/messagebus/messaging.go#L48

So by updating we can then use MQTT simply by updating the `trigger.Configuration.MessageBus` properties, which is passed in. Is my assumption correct?

Reply:
Correct and trigger.Configuration.MessageBus comes from the configuartion.toml